### PR TITLE
Child calendar

### PR DESCRIPTION
**Pull request by mikolajmirko**
(includes working logic from `CalendarCubit`)

### CODE REVIEW

File: i18n/en_US.json [hunk 1]
```diff
@@ -466,6 +466,15 @@
 					}
 				}
 			},
+			"calendar": {
+				"header": {
+					"title": "Plan calendar"
+				},
+				"content": {
+					"plansOnDateTitle": "Plans on {DATE}",
+					"noPlansOnDateTitle": "No plans on this day"
```
Comment (stasgora) [inline i18n/en_US.json:475]: for this day

File: i18n/en_US.json [hunk 2]
```diff
@@ -466,6 +466,15 @@
 					}
 				}
 			},
+			"calendar": {
+				"header": {
+					"title": "Plan calendar"
+				},
+				"content": {
+					"plansOnDateTitle": "Plans on {DATE}",
```
Comment (stasgora) [inline i18n/en_US.json:474]: Plans for

File: lib/utils/calendar_utils.dart [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import 'package:fokus/services/app_locales.dart';
+
+Widget buildCalendar(
+	CalendarController controller,
+	BuildContext context,
+	Map<Date, List<UIPlan>> events,
+	Function(DateTime, List<dynamic>) onDaySelected,
+	Function(DateTime, DateTime, CalendarFormat) onCalendarCreated,
+	Function(DateTime, DateTime, CalendarFormat) onVisibleDaysChanged,
+	CalendarBuilders builders
```
Comment (stasgora) [inline lib/utils/calendar_utils.dart:16]: Named args?

File: lib/pages/child/calendar_page.dart [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+								child: _buildPlanList()
+							)
+						]
+					)
+				]
+			)
+		);
+	}
+
+	Widget _buildPlanList(){
```
Comment (stasgora) [inline lib/pages/child/calendar_page.dart:78]: You can also extract this method from both pages to util

**Review by stasgora — CHANGES_REQUESTED** [review]

File: lib/pages/child/calendar_page.dart (hunk 4 above)
Comment (mikolajmirko) [reply lib/pages/child/calendar_page.dart:78]: With cubit inside? 🤔 
Also this chunk is different for both calendar pages:
obraz

So I can get the Segment widget out I guess, but there would be a lot of arguments.. (remember that texts for children can be changed later to better fit them).

Comment (stasgora) [reply lib/pages/child/calendar_page.dart:78]: Okay, lets stick with this than

**Review by stasgora — APPROVED** [review]

**Pull request merged**